submissions/kovalov/dom/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="main-header">
+      <button
+        class="main-header__button"
+        data-side-menu-button="open"
+      >
+        <svg class="main-header__button-icon">
+          <use href="./assets/sprite.svg#menu-icon"></use>
+        </svg>
+      </button>
+      <a href="./" class="main-header__title">Fantasy Planets</a>
+    </header>
+    <main class="main-section">
+      <div class="main-section__container" data-main-content>
+        <article class="main-section__content">
+          <p class="main-section__lead">
+            Planets in science fiction are fictional planets that
+            appear in various media of the science fiction genre as
+            story-settings or depicted locations.
+          </p>
+        </article>
+      </div>
+    </main>
+    <div class="side-menu side-menu--hidden" data-side-menu="closed">
+      <button class="side-menu__button" data-side-menu-button="close">
+        <svg class="side-menu__button-icon">
+          <use href="./assets/sprite.svg#close-icon"></use>
+        </svg>
+      </button>
+      <nav
+        class="side-menu__navigation navigation"
+        data-side-menu-content
+      >
+        <ul class="navigation__list">
+          <li class="navigation__list-item">
```

Comment:
Still not fixed [https://github.com/kottans/frontend-2022-homeworks/pull/20#discussion_r932934217](https://github.com/kottans/frontend-2022-homeworks/pull/20#discussion_r932934217)